addons/controller_icons/Settings.gd

```diff
@@ -18,7 +18,8 @@ enum Devices {
 	XBOX360,
 	XBOXONE,
 	XBOXSERIES,
-	STEAM_DECK
+	STEAM_DECK,
+	OXP,
```

Comment:
This is going to need to be a lot more specific. Different OXP devices have different setups

Reply:
Yeah, ultimately I want to update this so that it will see if there's a handheld platform with custom controller icons. If there is, it will use those.

This is related to the refactoring I did here where we can define the controller icons in the platform resource:
https://github.com/ShadowBlip/OpenGamepadUI/pull/225